web/modules/custom/nys_registration/src/RegistrationHelper.php

```diff
@@ -68,10 +78,10 @@ public function getDistrictFromAddress(array $address_parts): ?Term {
        * @var \Drupal\taxonomy\Entity\Term|null $district_term
        */
       $district_term = current(
-            $this->entityTypeManager
-              ->getStorage('taxonomy_term')
-              ->loadByProperties(['field_district_number' => $district])
-        ) ?: NULL;
+        $this->entityTypeManager
+          ->getStorage('taxonomy_term')
+          ->loadByProperties(['field_district_number' => $district])
```

Comment:
As you said, probably not an issue here, since there's no access applicability (all districts and senators are 100% public), but the distinction is noted.

Interestingly, I actually benchmarked three different approaches while crafting the existence check in `generateUserName()` - `loadByProperties()`, using a new entity query, and cloning a pre-made entity query.  The new query beat the others by a noticeable margin.  `loadByProperties()` was the worst, since it is loading the entire entity.  That is desired behavior here, but something I'll keep in mind as well.